container/sagemaker/python_service.py

```diff
@@ -157,6 +159,24 @@ def on_post(self, req, res):
                 'error': str(e)
             }).encode('utf-8')
 
+    def _read_model_config(self):
+        models = []
+        name_key = '    name: '
+        uri_key = '    base_path: '
+        with open(MODEL_CONFIG_FILE_PATH, 'r') as f:
+            line = f.readline()
+            while line:
+                if line.startswith(name_key):
+                    model_name = line[len(name_key) + 1:len(line) - 3]
```

Comment:
Same comment as above. I'm sure it's functional, but it triggers a lot of red flags in my mind.

Reply:
Same here, used regex search to read values from model-config.cfg